assets/queries/openAPI/paths_object_empty/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "815021c8-a50c-46d9-b192-24f71072c400",
+  "queryName": "Paths Object is Empty",
```

Comment:
The folder name is slightly different from the 'queryName' in the metadata file

Reply:
I think it's ok